modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -823,6 +828,11 @@ public void visit(BLangRetry retryNode) {
         result = retryNode;
     }
 
+    @Override
+    public void visit(BLangTypeAccessExpr accessExpr) {
```

Comment:
Let's rename this to BLangTypeofExpression

Reply:
+1